docroot/themes/humsci/humsci_basic/src/js/main-nav/nested-toggler.js

```diff
@@ -0,0 +1,72 @@
+const togglers = document.querySelectorAll('.hb-nested-toggler');
+const mobileNavBreakpoint = 992;
+let windowWidth = window.innerWidth;
+
+if (togglers) {
+  for (let i = 0; i < togglers.length; i += 1) {
```

Comment:
hmm I dont see this logic happening to all my nested-togglers. Only the first one it seems 🤔 maybe we can pair on this

Reply:
I updated the description with one config change that I had forgotten about. Let me know if this fixes things.